src/app/plan/components/Wizard/HooksStep.tsx

```diff
@@ -0,0 +1,220 @@
+import React, { useEffect, useState, useContext } from 'react';
+import {
+  Table,
+  TableBody,
+  TableHeader,
+} from '@patternfly/react-table';
+import { Spinner } from '@patternfly/react-core/dist/esm/experimental';
+import spacing from '@patternfly/react-styles/css/utilities/Spacing/spacing';
+import { PlusCircleIcon } from '@patternfly/react-icons';
+
+import {
+  Grid,
+  GridItem,
+  Text,
+  TextContent,
+  TextVariants,
+  Bullseye,
+  EmptyState,
+  EmptyStateVariant,
+  EmptyStateBody,
+  Title,
+  Button,
+  EmptyStateIcon,
+  InputGroup,
+  TextInput,
+  Flex,
+  FlexItem,
+  FlexModifiers
+} from '@patternfly/react-core';
+import { SearchIcon } from '@patternfly/react-icons';
+import {
+  AddEditMode,
+} from '../../../common/add_edit_state';
+import HooksFormContainer from './HooksFormContainer';
+
+const classNames = require('classnames');
+const styles = require('./HooksStep.module')
+
+const HooksStep = (props) => {
```

Comment:
We should define an interface for the props here, give this type annotations and destructure props in the function sig.

```ts
const HooksStep: React.FunctionComponent<IHooksStepProps> = ({
  ...
}: IHooksStepProps) => {
  ...
};
```

Reply:
Seems like a lot of the props here are the same as the hooks form components, so maybe those types could either be defined down in the form and extended here, or defined here and `Pick<...>`'d down in the form.

Reply:
Add this one to the list for future TS support